### ISSUE TITLE
Cannot remove data source

### ISSUE THREAD

**Issue by henare** [ciudadanointeligente/write-it#1132]
Even if it's broken there's no way for me to remove a data source. I guess I could tell WriteIt to never update it so maybe this isn't a massive issue.

![selection_004](https://cloud.githubusercontent.com/assets/48945/13241227/431e12c4-da3d-11e5-803e-e4708811bfac.png)

**Comment by tmtmtmtm** [comment]
This is essentially a duplicate of https://github.com/ciudadanointeligente/write-it/issues/725 (though that was written when the only possible data source was a PopIt)

**Comment by patcon** [comment]
Yep, I think this could be removed as dup if you update the title and body of the old issue to be more general (I wouldn't have known to click yours even if it had been in search results)

**Comment by patcon** [comment]
Also, you should add you :+1: over there @henare :)
